src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -46,6 +46,38 @@ public WebElement forElementPresent(By by) {
   }
 
   /**
+   * Checks if the element is clickable on browser
+   * <p/>
+   * * @param element The element to be checked
+   */
+  public WebElement forElementClickable(WebElement element) {
+    changeImplicitWait(0, TimeUnit.MILLISECONDS);
+    try {
+      element.getTagName();
+    } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)